From e0bd193a756235e3186b2b7464ee61025ff3e344 Mon Sep 17 00:00:00 2001
From: Tycho Andersen <tycho@tycho.ws>
Date: Sun, 18 Sep 2016 19:30:32 -0600
Subject: [PATCH] fix build with latest verison of optparse-applicative

Apparently it stopped exporting <>, so let's import that by hand.

Signed-off-by: Tycho Andersen <tycho@tycho.ws>
---
 generator/xcffibgen.hs | 1 +
 xcffib.cabal           | 3 ++-
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/generator/xcffibgen.hs b/generator/xcffibgen.hs
index 52d89a5..bbad4de 100644
--- a/generator/xcffibgen.hs
+++ b/generator/xcffibgen.hs
@@ -16,6 +16,7 @@
 module Main where
 
 
+import Data.Semigroup
 import Data.XCB.Types
 import Data.XCB.Python.Parse
 
diff --git a/xcffib.cabal b/xcffib.cabal
index 711cae3..8ee3222 100644
--- a/xcffib.cabal
+++ b/xcffib.cabal
@@ -54,7 +54,8 @@ executable xcffibgen
                  containers,
                  mtl >= 2.1,
                  attoparsec,
-                 bytestring
+                 bytestring,
+                 semigroups
   ghc-options: -Wall
 
 test-suite PyHelpersTests
